src/test/unit/parsers/test_utils.py

```diff
@@ -239,6 +239,15 @@ def test_recursive_json_join(d1, d2, result):
                 "detailType": "Scheduled Event",
             },
         ),
+        (  # elb example trigger
+            {
+                "httpMethod": "GET",
+                "resource": "resource",
+                "headers": {"Host": "www.google.com"},
+                "requestContext": {"elb": "Elb", "stage": "1", "requestId": "123"},
+            },
+            {"triggeredBy": "unknown"},
```

Comment:
What happens when we have trigger by unknown?

Reply:
![image](https://user-images.githubusercontent.com/50795205/72426339-34465980-3792-11ea-8ecf-7f8a4a16f224.png)